### ISSUE TITLE
Contract hierarchy

### ISSUE THREAD

**Issue by kaber2** [stader-labs/ethx#214]
Hello,

I'm wondering about the contract hierarchy. We have the config, which contains pointers to ETHx, the Oracle, ..., and we have all these entities, which can update their internal view of the config address.

This basically just loosely links these entities together, there's nothing preventing the config from pointing to lets say the Oracle, which can itself regard a different config as the authorative one.

Basically, I'm wondering what the "top level" element is, from which I can start following downwards the path.

Thanks!
